pkg/kubernetes/scripts/registry.js

```diff
@@ -163,6 +164,59 @@
                 return select().kind("Project");
             };
 
+            // TODO: invalidate cache on role changes
+            var sarCache = { };
+
+            $scope.subjectAccessReview = function subjectAccessReview(project, user, verb, resource) {
+                var result = false;
+
+                // if no project was given, check the first one as an approximation
+                // TODO: is that sufficient, or should we iterate over all
+                // projects? (much more expensive)
+                if (user && !project) {
+                    var projects = select().kind("Project");
+                    for (var p in projects) {
+                        project = projects[p].metadata.name;
+                        break;
+                    }
+                    if (!project)
+                        // we didn't init sufficiently far yet, defer to the
+                        // next call; or the user really has no projects
+                        return false;
```

Comment:
We should try to structure this so this doesn't happen, base it of an event of projects loading.